gobblin-core/src/main/java/gobblin/source/extractor/extract/sftp/SftpFsHelper.java

```diff
@@ -162,6 +175,21 @@ public void connect()
   }
 
   /**
+   * In a map reduce environment the method sets the private key from a file on HDFS
+   */
+  private void setIdentityFromDefaultFileSystem(String privateKey, JSch jsch) throws IOException, JSchException {
+    FSDataInputStream privateKeyStream = null;
+    try {
```

Comment:
https://github.com/linkedin/gobblin/pull/436 the agreement was to bump up to Java 7 right?